docs/source/l1_to_l2b_e2e.rst

```diff
@@ -0,0 +1,187 @@
+Process L1 to L2b data
+-----------------------
+
+This section demonstrates how process Level 1 (L1) data into Level 2b (L2b) data using the ``corgidrp`` package.
+
+The focus is on explaining the key steps involved in processing data, including setting up calibration files, running the data processing pipeline, and understanding the core steps for transforming raw data into processed output. The test can be run by processing raw L1 data into L2b data using the ``walker.walk_corgidrp`` function. This function is the core of the pipeline and is responsible for the actual data processing. Here is how you can set up and run the test:
+
+Calibration Files
+~~~~~~~~~~~~~~~~~
+
+To process the L1 data, you will need several calibration files. These files are used in the calibration process to adjust the raw data accordingly. The following calibration files are required:
+
+    - Noise Maps: Used to account for noise in the sensor data.
+    - Non-Linearity Calibration: Corrects for any non-linearities in the sensor's response.
+    - Dark Current Calibration: Accounts for any dark current noise in the sensor.
+    - Flat Field Calibration: Used to correct for uneven sensitivity across the detector.
+    - Bad Pixel Maps: Identifies pixels that are defective or not functioning properly.
+
+
+Test Setup
```

Comment:
I think we should rename this something else other than "Test Setup", since this isn't a test. 

Maybe we should have a section on "Setup" where we setup all the calibrationfiles we need and one section on "Pipeline Execution" that contains the call to walk_corgidrp()